### DISCUSSION TITLE
How can I integration with laravel tenancy?

### DISCUSSION THREAD

**Discussion by luckykenlin** [awcodes/filament-curator#495] [Q&A]
Hi everyone,

I'm seeking some assistance with integrating [Laravel Tenancy](https://github.com/archtechx/tenancy) into my project. I would appreciate any guidance or experience you could share.

I am facing two issues:

1. I have two panels: AdminPanel and TenantPanel. I only want to register Curator on TenantPanel, but I encounter the exception "Plugin [awcodes/curator] is not registered for admin panel." As a temporary solution, I have also registered Curator on AdminPanel and disabled Navigation Register.

2. Uploading files works fine, but the preview image is broken, and I cannot use the `<x-curator-glider />` component.

Thank you in advance for any help or advice you can provide!

**Comment by luckykenlin** [comment]
I tried to use cloud disk, everything Is working fine.  I have no idea, how I can make these 2 packages work together on local or public disk.   

if anyone can share ideas or tips are really appreciate

**Comment by awcodes** [comment]
If you can give a reproduction repo I'm happy to explore it, but there  is currently a bug in filament with plugin registration. But this might go beyond that. But I'll need a repo to nail it down.

**Reply by luckykenlin** [reply]
https://github.com/luckykenlin/filament-curator-with-laravel-tenancy

Thanks @awcodes
